### ISSUE TITLE
[Feature Request] Highlight currently played song

### ISSUE THREAD

**Issue by reinerh** [ampache/ampache#2697]
## Describe what you don't like about Ampache, or what it is missing

When I play a song and I'm currently on the album page that contains this song, it's not immediately obvious which song of the album is being played.
I have to look down to the player to find the song title, then scan the track list of the album to find it.

## Describe how you would like the feature to work**

When being on the page with the currently played song, the whole row should be highlighted, for example surrounding it with a thick border.
When the song changes, it should also automatically update the highlight.

**Comment by lachlan-00** [comment]
so in the ui rows like this? (but obviously not the same as hovering)
![image](https://user-images.githubusercontent.com/1305249/104718893-5db80d80-5777-11eb-8a20-f48040eb2688.png)

there is definitely a now_playing which means we can make it usable in the ui if it's needed

**Comment by reinerh** [comment]
Yes, something like this. Maybe a little more subtle than the hovering highlight.
Other players paint a border around the currently played track, or use a bold or differently colored font.
But I don't have a strong opinion on how it will be highlighted.

**Comment by usox** [comment]
@reinerh Maybe a small animated "spectrum" icon like this one https://www.iconfinder.com/icons/795990/audio_music_spectrum_visualizer_icon

**Comment by reinerh** [comment]
@usox That would also work! :)
